com.ibm.streamsx.inet/impl/java/src/com/ibm/streamsx/inet/rest/servlets/ReqWebMessage.java

```diff
@@ -15,11 +10,17 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+/**
+* Licensed Materials - Property of IBM
+* Copyright IBM Corp. 2017 
+* @author mags
```

Comment:
Please remove `@author` tags, they tend to be avoided in open source projects to avoid implicit code ownership claims.